crates/aptos/src/move_tool/mod.rs

```diff
@@ -909,7 +909,6 @@ impl IncludedArtifacts {
         experiments.append(&mut move_options.experiments.clone());
         experiments.append(&mut more_experiments);
 
-        // TODO(#14441): Remove `None |` here when we update default CompilerVersion
```

Comment:
This comment is removed because `LATEST_STABLE_COMPILER_VERSION` is set as the default value cc @brmataptos @vineethk